src/test/java/com/stripe/model/AccountCreateParamsTest.java

```diff
@@ -0,0 +1,90 @@
+package com.stripe.model;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+import org.junit.Test;
+
+public class AccountCreateParamsTest {
+  @Test
+  public void testBuilderCreateParams() {
+    AccountCreateParams.Builder builder = AccountCreateParams.builder();
+    AccountCreateParams createParams = builder
+        .beginLegalEntity()
```

Comment:
For me this is an odd builder. The manual indentation here makes it easy to understand in this review, but I think a typical automatic code formatter will put these at the same indentation level, making this really difficult to understand visually.

I would probably just go with a protobuf style builder (with lots of parentheses) even though it's more verbose.

Reply:
+1 as well to the protobuf pattern for the visualization benefits. Especially in the future world of PaymentIntents, it seems like more Stripe APIs have requests that are nicely broken down into logical subhashes. Having a separate subbuilder for each logical subhash makes it easy to visualize the requests and also to have code reuse as people move from Charges -> PaymentIntents etc.

Reply:
ack! thank you everyone for the feedback